packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -334,10 +334,11 @@ Adds a new permission to an existing Role
 
 #### Request parameters
 
-| Parameter      | Description                          | Param Type   | DataType                              | Required |
-| :------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
-| `roleIdOrLabel`  | `id` or `label` of the Role          | URL          | String                                | TRUE     |
+| Parameter        | Description                          | Param Type   | DataType                             | Required |
+| :----------------| :------------------------------------| :----------- | :------------------------------------| :------- |
+| `roleIdOrLabel`  | `id` or `label` of the Role          | URL          | String                               | TRUE     |
 | `permissionType` | Permission to add to the Role        | URL          | [Permission](#permission-types) name | TRUE     |
+| `conditions` <ApiLifecycle access="ea" />      | Conditions for further restricting a permission     | Body         | [Condition](#condition-object) object | FALSE    |
```

Comment:
Just a question do we want to document that some attributes may not be able to be scoped for some permissions e.g. firstName, lastName, etc.

Reply:
Good point. I will add a note here.

Reply:
Actually, it is already listed here: https://help.okta.com/oie/en-us/Content/Topics/Security/custom-admin-role/permission-conditions.htm.
I will can link to this page from the ConditionObject section.